src/expandableCalendar/index.js

```diff
@@ -207,13 +207,13 @@ class ExpandableCalendar extends Component {
   }
 
   getMarkedDates() {
-    const {context, markedDates} = this.props;
+    const {context, markedDates, disabledByDefault} = this.props;
```

Comment:
You didn't declare the prop and described it. The prop's name is unclear.

Reply:
Can we check if the `marked[context.date]` is disabled before we apply this prop?

Reply:
Sorry I don't get you. This prop is inherited from Calendar that should be false by default. If that prop is true we don't want to go in the else block down in the code. If that prop is somehow undefined the behaviour remain as is. If you think something could be improved could you please include suggestions code or contribute to this branch? Thabks

Reply:
Yes, you're right, but that prop is doing something else and you are adding an extra behaviour on it. It's a breaking change for those who use this prop and don't want your extra behaviuor. Also, I run the code and it doesn't do anything, the date is still being automatically selected when you move to the previous/next month. What am 
I missing?

Reply:
I see. Thanks for the explanation. However I did this months ago and I am not able to contribute at the moment so I don't know if something has changed meanwhile. I thought this was a bug but since a new prop is required this feature would be much appreciated. Many thanks